alpha/action/render.go

```diff
@@ -51,6 +51,7 @@ var ErrNotAllowed = errors.New("not allowed")
 type Render struct {
 	Refs           []string
 	Registry       image.Registry
+	RegOptions     []containerdregistry.RegistryOption
```

Comment:
In general, the idea has always been that as a library user you get one of:
1. a default registry with no customization
2. your own registry, customized however you want

If you want 1), you set `Registry` to `nil`
If you want 2), you set `Registry` to your custom registry

This change would muddy those waters because `RegOptions` would only be used if `Registry` is `nil`.

Perhaps we could move `createRegistry` to some `internal` package that the CLI code has access to and then have all the relevant CLIs call that utility function so that they can consistently set the `Registry` field?